spring-integration-core/src/main/java/org/springframework/integration/store/SimpleMessageStore.java

```diff
@@ -58,6 +58,12 @@
 	private volatile boolean isUsed;
 
 	private volatile boolean copyOnGet = false;
+	
+	private final int maxWaitInMilliseconds;
```

Comment:
Well, we always can address the `Milliseconds` concern in the JavaDocs.
Really I don't see reason to overhead it...